client/reader/stream/utils.js

```diff
@@ -68,6 +68,7 @@ export function combine( postKey1, postKey2 ) {
 	return combined;
 }
 
+//@TODO: operate on things with dates...
```

Comment:
no idea what i was talking about here...maybe i already fixed it since CCes work